### PR TITLE
manpage: update missing document

### LINKED ISSUE #38351: Switch is missing from the man page
`man brew` doesn't list the `switch` command in its output.

Running homebrew 0.9.5.

### PR DESCRIPTION
**Pull request by xu-cheng**
Closes #38351.

### CODE REVIEW

File: Library/Homebrew/manpages/brew.1.md [hunk 1]
```diff
@@ -333,6 +342,9 @@ Note that these flags should only appear after a command.
     in a Xcode-only configuration since it adds tools like make to your PATH
     which otherwise build-systems would not find.
 
+  * `switch` <name> <version>:
+    Symlink keg <name> with specific <version>.
```
Comment (mikemcquaid) [inline Library/Homebrew/manpages/brew.1.md]: Might want to elaborate a bit on `symlink` here.

**Comment by mikemcquaid** [comment]
Great work :clap:

File: Library/Homebrew/manpages/brew.1.md (hunk 1 above)
Comment (xu-cheng) [inline Library/Homebrew/manpages/brew.1.md]: Updated to `Symlink all of the specific <version> of keg <name>'s install to Homebrew prefix.`

File: Library/Homebrew/manpages/brew.1.md [hunk 2]
```diff
@@ -333,6 +342,9 @@ Note that these flags should only appear after a command.
     in a Xcode-only configuration since it adds tools like make to your PATH
     which otherwise build-systems would not find.
 
+  * `switch` <name> <version>:
+    Symlink all of the specific <version> of keg <name>'s install to Homebrew prefix.
```
Comment (mikemcquaid) [inline Library/Homebrew/manpages/brew.1.md]: Sounds good except I'd remove the use of `keg` as it's a bit jargony.

**Pull request closed without merging**